src/Microsoft.ApplicationInsights.AspNetCore/Implementation/TelemetryConfigurationOptionsSetup.cs

```diff
@@ -51,9 +52,9 @@ public void Configure(TelemetryConfiguration configuration)
 
             if (this.telemetryProcessorFactories.Any())
             {
-                foreach (Func<ITelemetryProcessor, ITelemetryProcessor> processorFactory in this.telemetryProcessorFactories)
+                foreach (ITelemetryProcessorFactory processorFactory in this.telemetryProcessorFactories)
```

Comment:
Snapshot Collector ("exception exemplification") doesn't reference this repo. It references the base repo. This won't be a breaking change for us. We'll just have to update our docs.